### DISCUSSION TITLE
Unable to disable Snacks toggle

### DISCUSSION THREAD

**Discussion by dniets** [folke/snacks.nvim#309] [Q&A] [answered]
I have been using LazyVim for a long time and since [`v14.X`](https://www.lazyvim.org/news) it seems that [`toggle` has been enabled by default](https://www.lazyvim.org/plugins/ui#snacksnvim)

I have been trying to [disable toggle](https://github.com/folke/snacks.nvim/blob/main/docs/toggle.md) to have the old flavor of [`which-key`](https://github.com/folke/which-key.nvim) with the `preset = "classic"`, but I'm so stupid that I'm unable to do so

Does anyone knows how to do so?

**Comment by folke** [comment, marked as answer]
toggle has nothing to do with the which-key preset.

To change the preset `{"which-key.nvim", opts = {preset = "classic"}}`

**Comment by dniets** [comment]
ok i'm actually retard 😞 

it's seems that the default preset has changed with latest version

sorry and thanks @folke
